deploy/templates/nstemplatetiers/advanced-code.yaml

```diff
@@ -15,6 +15,18 @@ objects:
     labels:
       provider: codeready-toolchain
     name: ${USERNAME}-code
+- apiVersion: v1
+  kind: RoleBinding
+  metadata:
+    labels:
+      provider: codeready-toolchain
+    name: codeready-toolchain-code
```

Comment:
From my point of view, `codeready-toolchain-code` name doesn't bring much information - we know that it's inside of the `code` namespace and using the label we also know about the provider `codeready-toolchain`. 
I would rather name it `user-admin`, `user-edit` or `edit-code` to highlight the purpose of the resource (`RoleBinding`)

Reply:
Good point

Reply:
Done in https://github.com/codeready-toolchain/host-operator/pull/106/commits/d17c13c1e5c0d47270b843b45f7e33389089bfce